README.md

```diff
@@ -17,6 +17,7 @@ This is the repository for the Open Source Vulnerability schema (OSV Schema), wh
 - [PyPI Advisory Database](https://github.com/pypa/advisory-database)
 - [Python Software Foundation Database](https://github.com/psf/advisory-database)
 - [RConsortium Advisory Database](https://github.com/RConsortium/r-advisory-database)
+- [Red Hat](https://access.redhat.com/security/data)
```

Comment:
Looks like this should be what it is (at least it works and is consistent with docs/schema.md)
```suggestion
- [Red Hat](https://security.access.redhat.com/data)
```

Reply:
Fixed, thanks.